src/player.js

```diff
@@ -202,53 +220,49 @@ export default class Player extends FakeEventTarget {
    * @returns {void}
    */
   _configureOrLoadPlugins(plugins: Object = {}): void {
-    Object.keys(plugins).forEach((name) => {
-      // If the plugin is already exists in the registry we are updating his config
-      const plugin = this._pluginManager.get(name);
-      if (plugin) {
-        plugin.updateConfig(plugins[name]);
-        this._config.plugins[name] = plugin.getConfig();
-      } else {
-        // We allow to load plugins as long as the player has no engine
-        if (!this._engine) {
-          this._pluginManager.load(name, this, plugins[name]);
-          let plugin = this._pluginManager.get(name);
-          if (plugin) {
-            this._config.plugins[name] = plugin.getConfig();
-            if (typeof plugin.getMiddlewareImpl === "function") {
-              this._playbackMiddleware.use(plugin.getMiddlewareImpl());
+    if (plugins) {
+      Object.keys(plugins).forEach((name) => {
```

Comment:
Yes, it can be empty.
Pay attention that the received arg of this method is the **new** plugins that received from the configure and for that it can be null.